File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/ordprintdto/d/OrdPrintDataDTO.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using iih.ci.ord.ordprn.d;

namespace iih.ci.iih.ci.ord.dto.ordprintdto.d
{
    /// <summary>
    /// <para>描    述 : </para>
    /// <para>说    明 : </para>
    /// <para>项目名称 :  iih.ci.iih.ci.ord.dto.ordprintdto.d    </para>    
    /// <para>类 名 称 :  OrdPrintDataDTO					</para> 
    /// <para>版 本 号 :  v1.0.0.0</para> 
    /// <para>作    者 :  hums</para> 
    /// <para>修 改 人 :  hums</para> 
    /// <para>创建时间 :  2016/8/22 11:34:10</para>
    /// <para>更新时间 :  2016/8/22 11:34:10</para> 
    /// <para>Copyright @ 北大医信（IIH项目组） 2016. All rights reserved.</para> 
    /// </summary>
    public class OrdPrintDataDTO : OrdPrintDO
    {

        /// <summary>
        /// 服务名
        /// </summary>
        public string Name_srv
        {
            get { return getAttrVal<string>("Name_srv", null); }
            set { setAttrVal<string>("Name_srv", value); }
        }

        /// <summary>
        /// 剂量
        /// </summary>
        public string Quan_medu
        {
            get { return getAttrVal<string>("Quan_medu", null); }
            set { setAttrVal<string>("Quan_medu", value); }
        }

        /// <summary>
        /// 剂量单位
        /// </summary>
        public string Medu_name
        {
            get { return getAttrVal<string>("Medu_name", null); }
            set { setAttrVal<string>("Medu_name", value); }
        }

        /// <summary>
        /// 物品名称
        /// </summary>
        public string Name_mm
        {
            get { return getAttrVal<string>("Name_mm", null); }
            set { setAttrVal<string>("Name_mm", value); }
        }

        /// <summary>
        /// 物品规格
        /// </summary>
        public string Spec
        {
            get { return getAttrVal<string>("Spec", null); }
            set { setAttrVal<string>("Spec", value); }
        }

        /// <summary>
        /// 用法名称
        /// </summary>
        public string Route_name
        {
            get { return getAttrVal<string>("Route_name", null); }
            set { setAttrVal<string>("Route_name", value); }
        }

        /// <summary>
        /// 频次名称
        /// </summary>
        public string Freq_name
        {
            get { return getAttrVal<string>("Freq_name", null); }
            set { setAttrVal<string>("Freq_name", value); }
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/ordprintdto/d/OrdPrintDataDTO.java
package iih.ci.ord.dto.ordprintdto.d;

import iih.ci.ord.ordprn.d.OrdPrintDO;

/**
 * 医嘱打印数据DTO
 * 注：此DTO没使用元数据管理器
 * @author hums
 *
 */
public class OrdPrintDataDTO extends OrdPrintDO {
	
	private static final long serialVersionUID = 1L;

	/**
	 * 服务名
	 * @return String
	 */
	public String getName_srv() {
		return ((String) getAttrVal("Name_srv"));
	}
	/**
	 * 服务名
	 * @param Name_srv
	 */
	public void setName_srv(String Name_srv) {
		setAttrVal("Name_srv", Name_srv);
	}
	/**
	 * 剂量
	 * @return String
	 */
	public String getQuan_medu() {
		return ((String) getAttrVal("Quan_medu"));
	}
	/**
	 * 剂量
	 * @param Quan_medu
	 */
	public void setQuan_medu(String Quan_medu) {
		setAttrVal("Quan_medu", Quan_medu);
	}
	/**
	 * 剂量单位
	 * @return String
	 */
	public String getMedu_name() {
		return ((String) getAttrVal("Medu_name"));
	}
	/**
	 * 剂量单位
	 * @param Medu_name
	 */
	public void setMedu_name(String Medu_name) {
		setAttrVal("Medu_name", Medu_name);
	}
	/**
	 * 物品名称
	 * @return String
	 */
	public String getName_mm() {
		return ((String) getAttrVal("Name_mm"));
	}
	/**
	 * 物品名称
	 * @param Name_mm
	 */
	public void setName_mm(String Name_mm) {
		setAttrVal("Name_mm", Name_mm);
	}
	/**
	 * 物品规格
	 * @return String
	 */
	public String getSpec() {
		return ((String) getAttrVal("Spec"));
	}
	/**
	 * 物品规格
	 * @param Spec
	 */
	public void setSpec(String Spec) {
		setAttrVal("Spec", Spec);
	}
	/**
	 * 用法名称
	 * @return String
	 */
	public String getRoute_name() {
		return ((String) getAttrVal("Route_name"));
	}
	/**
	 * 用法名称
	 * @param Route_name
	 */
	public void setRoute_name(String Route_name) {
		setAttrVal("Route_name", Route_name);
	}
	/**
	 * 频次名称
	 * @return String
	 */
	public String getFreq_name() {
		return ((String) getAttrVal("Freq_name"));
	}
	/**
	 * 频次名称
	 * @param Freq_name
	 */
	public void setFreq_name(String Freq_name) {
		setAttrVal("Freq_name", Freq_name);
	}

	public Object getAttrVal(String attrName) {
		Object r = this.propStoreMap.get(attrName);
		return r;
	}
}
